README-docker-compose.md

```diff
@@ -222,7 +226,9 @@
     - _NOTE: CTRL-c will exit_
 - Run migrations:
   - After creating migrations, resetting your database, or starting on a fresh install you will need to run migrations to make the needed changes to database. This command looks at the migrations on disk and compares them to the list of migrations in the `django_migrations` database table and runs any migrations that have not been run.
-    - `docker-compose run --rm web python3 manage.py migrate`
+    - `docker-compose run --rm web python3 manage.py migrate` To run `osf` migrations
+    - `docker-compose run --rm gv python manage.py migrate` To run `gravyvalet(gv)` migrations
+  - After creating migrations, resetting your database, or starting on a fresh install you will need to run migrations to make the needed changes to database. This command looks at the migrations on disk and compares them to the list of migrations in the `django_migrations` database table and runs any migrations that have not been run.
```

Comment:
nit: this line is duplicated (tho i agree it makes more sense after the `migrate` commands